src/Providers/DataGrid/Wijmo/Features/Styling.ts

```diff
@@ -25,8 +25,8 @@ namespace Providers.DataGrid.Wijmo.Feature {
 			let classList = [];
 			const column = this._grid.getColumn(columnID);
 
-			if (column) {
-				const classListString = column.provider.cssClassAll || ' ';
+			if (column && column.provider.cssClassAll) {
+				const classListString = column.provider.cssClassAll;
```

Comment:
This const could be created before the `if`, no?

Reply:
Well, it could, but it isused inside the if only, so I don't see why.

Reply:
not really, since you're using it in the condition! :P

Reply:
not very relevant